api/Request.json

```diff
@@ -1211,10 +1211,10 @@
               "version_added": false
             },
             "safari": {
-              "version_added": false
+              "version_added": "10.1"
```

Comment:
10.1 and 10.3 matches when `fetch()` is said to have shipped, which makes me quite confident this is correct: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API#Browser_compatibility